app/src/main/java/com/github/geohunt/app/ui/components/tutorial/Tutorial.kt

```diff
@@ -0,0 +1,240 @@
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(12.dp)
+    ) {
+        // Button to go to the previous tutorial slide
+        if (pageState.currentPage != 0) {
+            Button(
+                onClick = {
+                    if (pageState.currentPage > 0) scope.launch {
+                        pageState.scrollToPage(pageState.currentPage - 1)
+                    }
+                },
+                modifier = Modifier
+                    .align(Alignment.CenterStart)
+                    .testTag("Go back button"),
+            ) {
+                Icon(
+                    imageVector = Icons.Outlined.KeyboardArrowLeft,
+                    contentDescription = "Go back button"
+                )
+            }
+        }
+
+        // Button to skip the tutorial and go to the login screen
+        TextButton(
+            onClick = {
+                activity.replaceActivity(i = Intent(activity, LoginActivity::class.java))
+            },
+            modifier = Modifier
+                .align(Alignment.CenterEnd)
+                .testTag("Skip button"),
+        ) {
+            Text(
+                text = "Skip",
+                color = MaterialTheme.colors.onBackground,
+                fontSize = 16.sp,
+            )
+        }
+    }
+}
+
+/**
+ * Displays the button for moving to the next tutorial slide
+ * at the bottom of the tutorial screen
+ *
+ * @param activity The current activity
+ * @param pageState The state of the pager
+ * @param scope The coroutine scope
+ * @param slidesCount The size of the tutorial slides
+ * @param index The index of the current tutorial slide
+ * @param onButtonClick The function to call when the button is clicked
+ */
+@Composable
+fun BottomButtons(
+    activity: ComponentActivity,
+    pageState: PagerState,
+    scope: CoroutineScope,
+    slidesCount: Int,
+    index: Int,
+    onButtonClick: () -> Unit = {}
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(12.dp)
+    ) {
+        FloatingActionButton(
+            onClick = {
+                if (pageState.currentPage + 1 < slidesCount)
+                    scope.launch { pageState.scrollToPage(pageState.currentPage + 1) }
+                else activity.replaceActivity(Intent(activity, LoginActivity::class.java))
+            },
+            containerColor = Color.Black,
+            modifier = Modifier
+                .align(Alignment.CenterEnd)
+                .clip(RoundedCornerShape(15.dp, 15.dp, 15.dp, 15.dp))
+                .testTag("Go forward button"),
+        ) {
+            Icon(Icons.Outlined.KeyboardArrowRight,
+                tint = Color.White,
+                contentDescription = "Go forward button",
+            )
+        }
+    }
+}
+
+/**
+ * Displays the content of a list of tutorial slides
+ *
+ * @param items: The list of tutorial slides
+ */
+@Composable
+fun TutorialSlideContent(items: TutorialSlides) {
```

Comment:
Documentation doesn't match with the method and items should be renamed item.

Reply:
I have updated the documentation and corrected the name 'items' to 'slide'